### PR TITLE
Bump version on master in preparation for 0.2 release

### PR DESCRIPTION
**Pull request by thomkeh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 # We follow Semantic Versioning (https://semver.org/)
 _MAJOR_VERSION = "0"
-_MINOR_VERSION = "1"
-_PATCH_VERSION = "1"
+_MINOR_VERSION = "3"
```
Comment (olliethomas) [inline setup.py:7]: ?

Comment (thomkeh) [reply setup.py:7]: That's so that master is always ahead of any release.

Comment (thomkeh) [reply setup.py:7]: If master were `0.2.0.dev`, then it would be considered older than `0.2.0`.

Comment (olliethomas) [reply setup.py:7]: :D

**Pull request merged**